contracts/src/periphery/EigenDACertVerifierV1.sol

```diff
@@ -0,0 +1,152 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {IEigenDACertVerifierV1} from "src/periphery/interfaces/IEigenDACertVerifierV1.sol";
+import {IEigenDAThresholdRegistry} from "src/interfaces/IEigenDAThresholdRegistry.sol";
+import {IEigenDABatchMetadataStorage} from "src/interfaces/IEigenDABatchMetadataStorage.sol";
+import {EigenDACertVerificationV1Lib as CertLib} from "src/periphery/libraries/EigenDACertVerificationV1Lib.sol";
+import "src/interfaces/IEigenDAStructs.sol";
+
+/**
+ * @title CertVerifierV1 - EigenDA V1 certificate verification
+ * @author Layr Labs, Inc.
+ * @notice Contains all V1-specific verification functionality
+ */
+contract EigenDACertVerifierV1 is IEigenDACertVerifierV1 {
+    /// @notice Thrown when there is a length mismatch
+    error LengthMismatch();
+
+    /// @notice The EigenDAThresholdRegistry contract address
+    IEigenDAThresholdRegistry public immutable eigenDAThresholdRegistryV1;
```

Comment:
@samlaf #1472 

I personally prefer not to include comments like this that just restate the information already available in the variable name/type. wdyt?

I can agree with that. I'll cut out the obvious ones.

Reply:
I can agree with that. I'll cut out the obvious ones.

Reply:
https://github.com/Layr-Labs/eigenda/pull/1477/commits/96fda95576fc07c3045029a01d4df814b6eff3f8